client/src/components/Users/index.js

```diff
@@ -0,0 +1,4 @@
+import UserListItem from './UserListItem';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Can you add a styles.js to the same folder to avoid this and also setup same pattern as other components.

Reply:
it currently has no style, do we need a style sheet even when it has no style?

Reply:
it's probably not a terrible idea even if it is an empty dict